www/pages/rapidsMap.php

```diff
@@ -1,145 +1,353 @@
-</script>
-
+<?php
+include '../services/MapService.class.php';
+
+$mapData = new MapService();
+$allPinInfo = $mapData -> getAllMapPinInfo();
+$markers = $mapData -> generateMarkers($allPinInfo);
+
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+    
+    <link rel="stylesheet" href="css/maps.css" type="text/css">
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	
+	<title> Rapids Cemetery Map </title>
+
+	<link rel="stylesheet" href="css/font-awesome.css" type="text/css">
+	<link href="css/bootstrap.min.css" rel="stylesheet"/>
+	<script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script> 
+$(document).ready(function(){
+    $("#flip").click(function(){
+        $("#panel").slideToggle("slow");
+    });
+});
+$("#flip").on('click',function(){
+    $(this).children('i.fa-sort-down').toggleClass('i.fa-sort-up');
+});
+
+function openup() {
+  $(".popup-overlay, .popup-content").show("fast");
+}
+function shutdown() {
+  $(".popup-overlay, .popup-content").hide("fast");
+}
+</script>
+<style>
+.popup-overlay{
+   z-index:999;
+position: absolute;
+  background:#ffffff;
+  
+  width:100%;
+  height:100%;
+  left:0%; 
+  top:0%;
+}
+.popup-overlay.active{
+  /*displays pop-up when "active" class is present*/
+  visibility:visible;
+  text-align:center;
+}
+.popup-content.active {
+  /*Shows pop-up content when "active" class is present */
+  visibility:visible;
+}/*
+.gm-style-iw {
+	width: 500px !important;
+	height: 500px;
+	top: 15px !important;
+	left: 0px !important;
+	background-color: #fff0;
+	box-shadow: 0 1px 6px rgba(178, 178, 178, 0.6);
+	border: 1px solid rgba(72, 181, 233, 0.6);
+	border-radius: 2px 2px 10px 10px;
+}*/
+</style>
+<style> 
+#panel, #flip {
+    padding: 5px;
+    text-align: center;
+    background-color: #e5eecc;
+    border: solid 1px #c3c3c3;
+}
+
+#panel {
+    display: none;
+}
+</style>
+<style>
+.secondmenu input[type="radio"] {
+    opacity: 0;
+}
+.paragraph{margin: 80px 0px 10px!important;}
+</style>
+</head>
+<body>
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap.min.css" rel="stylesheet">
+    <script src="//code.jquery.com/jquery.js"></script>
+    <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js" type="text/javascript"></script>
+	<script type="text/javascript">
+        var markers = [
+    {
+        "title": 'Civil War',
+        "lat": '43.129465',
+        "lng": '-77.639334',
+        "description": '<img src=images/download.jpg style=width:100px;height:100px;/></br><h4>Robert Thomas</br><h6>(1829 - 1879)</h6></h4>Son of William Thomas,first Candlestick marker in Rochester,Robert was succeeded by his son Charles and help',

```

Comment:
This is not how the data is set up therefore this is invalid.
It needs to follow the same attributes that our db has. 
We do not return JSON in this application.
HTML is never saved in a DB so that would never be returned.